apps/frontend/src/styles/tailwindcss/components.css

```
@@ -10,7 +10,7 @@
 
     .header-link,
     .header-link:visited {
-      @apply font-body normal-case no-underline text-sov-white text-base transition;
+      @apply font-montserrat normal-case no-underline text-sov-white text-base transition;
```

```suggestion
      @apply normal-case no-underline text-sov-white text-base transition;
```

Please test it, but I think we can just remove the font setting here and on line 30, or change to `font-body` (we should be using Roboto for everything, unless I missed this somewhere in designs)

---

It was set to montserrat for header so I think it's correct (font-body was montserrat). If it should not be a montserrat, then font should be removed from the project entirely as it's not used anywhere else

---

Ok let's leave it and we'll remove it in another story, thanks